apps/super-app/src/routes/SupportDetailPage.tsx

```diff
@@ -13,14 +14,29 @@ import {
 } from '../services';
 import { Container } from '../common/components';
 import { PrimaryButton } from '../common/components/RoundedButton';
-import './SupportDetailWip.css';
+import './SupportDetailPage.css';
+import { Path } from './paths';
 
 function Header(props: { title: string }) {
   const { title } = props;
+  const navigate = useNavigate();
   return (
     <nav className="support-header">
-      <ArrowBackIosNewIcon />
-      <Typography variant="h4">{title}</Typography>
+      <IconButton
+        aria-label="back"
+        sx={{ color: 'text.primary' }}
+        onClick={() => navigate(Path.LandingPage)}
```

Comment:
Did we change course on the navigation?

Reply:
Yes we did. There's a new mock. Let's take the alignment for this one later, but get the logic out first.